stock_manual_shipping_weight/security/ir.model.access.csv

```diff
@@ -1,2 +1,2 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_edit_shipping_weight_wizard,access_edit_shipping_weight_wizard,model_edit_shipping_weight_wizard,account.group_account_user,1,1,1,1
+access_edit_shipping_weight_wizard,access_edit_shipping_weight_wizard,model_edit_shipping_weight_wizard,base.group_user,1,1,1,1
```

Comment:
since it is a warehouse functionality. The minimum warehouse security group would have to be set. Change that please.